static/module.json

```
@@ -2,7 +2,7 @@
   "id": "yze-combat",
   "title": "Year Zero Engine: ⚔️ Combat",
   "description": "Adds YZE combat mechanics to a game system: Card initiative, slow & fast actions, groups, and more!",
-  "version": "1.4.0",
+  "version": "1.5.0",
   "compatibility": {
     "minimum": 13.341,
     "verified": 13.345,
```

```suggestion
  "version": "1.4.0",
  "compatibility": {
    "minimum": 13.341,
    "verified": 13.347,
```
Reverting version number change as this is done at release.
Verified under 347 ✔️